docker-accumulo/accumulo-conf/accumulo-site.xml

```diff
@@ -127,7 +127,7 @@
 
   <property>
     <name>tserver.walog.max.size</name>
-    <value>256M</value>
+    <value>100M</value>
```

Comment:
Oh, actually, I just replaced the file - there were no changes. It is the original file from conf/examples/512MB/standalone/*

All values are by default.